README.md

```diff
@@ -96,25 +113,57 @@ fundamental differences in quality and design:
 To learn more about our correctness tests, please see the [Vector test harness][urls.vector_test_harness].
 
 
-## The Little Details Make All Of The Difference
+## The Little Details
 
-* **Zero delay start** - Starts and restarts without a delay.
+#### Data Model
+
+* **All data types** - [Logs][docs.data-model.log], [metrics][docs.data-model.metric], and traces (coming soon).
+* **Customizable log schema** - [Change Vector's log schema][docs.global-options#log_schema] to anything you like.
+* **Rich type system** - Support for [JSON primitive types][docs.data-model.log#types] and [timestamps][docs.data-model.log#timestamps].
+* **Metrics interoperability** - A [sophisticated metrics data model][docs.data-model.metric] ensures _correct_ interoperality between systems.
+* **Metrics aggregation** - Aggregated [histgorams][docs.data-model.metric#aggregated_histogram] and [summaries][docs.data-model.metric#aggregated_summary] allow for aggreation without loss of precision.
+
+#### Data Flow
+
+* **Pipelining** - A [directed acyclic graph processing model][docs.configuration] allows for flexible topologies.
+* **Control-flow** - Transforms like the [`swimlanes` transform][docs.transforms.swimlanes] allow for complex control-flow logic.
+* **Dynamic partitioning** - Create dynamic partitions on the fly ([ex][docs.sinks.aws_s3#partitioning]) with Vector's [templating syntax][docs.reference.templating].
+
+#### Data Processing
+
+* **Multi-line merging** - [Merge multi-line logs][docs.sources.file#multiline] into one event, such as stacktraces.
```

Comment:
I'd probably move this down the list a bit. The others are more interesting, core features.